test/e2e/reconciler/ingress/controller_test.go

```diff
@@ -158,30 +153,25 @@ func TestIngressController(t *testing.T) {
 				metav1.GroupResource{Group: apiresource.GroupName, Resource: "negotiatedapiresources"},
 			)
 			require.NoError(t, err)
-			kubefixtures.Create(t, sourceCrdClient.ApiextensionsV1().CustomResourceDefinitions(),
-				metav1.GroupResource{Group: "core.k8s.io", Resource: "services"},
-				metav1.GroupResource{Group: "apps.k8s.io", Resource: "deployments"},
-				metav1.GroupResource{Group: "networking.k8s.io", Resource: "ingresses"},
-			)
-
-			resources := sets.NewString("ingresses.networking.k8s.io", "deployments.apps", "services")
-			syncerFixture := framework.NewSyncerFixture(t, resources, source, orgClusterName, clusterName)
-
-			sink := syncerFixture.RunningServer
-			sinkConfig := sink.DefaultConfig(t)
-			sinkCrdClient, err := apiextensionsclientset.NewForConfig(sinkConfig)
-			require.NoError(t, err, "failed to create apiextensions client")
-
-			// TODO(jmprusi): Remove this one once Kind e2e is a thing.
-			t.Logf("Installing test CRDs into sink cluster...")
-			kubefixtures.Create(t, sinkCrdClient.ApiextensionsV1().CustomResourceDefinitions(),
+			err = configcrds.CreateFromFS(ctx, sourceCrdClient.ApiextensionsV1().CustomResourceDefinitions(), embeddedResources,
```

Comment:
CRD installation into the source cluster can be removed. The syncer's api importer will take care of that.